src/components/charts/TemperatureChart.vue

```diff
@@ -0,0 +1,1062 @@
+<template>
+  <div class="outer">
+    <div class="wrapper">
+      <div class="inner">
+        <div id="chart">
+
+          <div class="toolbar">
+            <button  id="one_month" @click="updateData('one_month')"
+              :class="{ active: selection === 'one_month' }"
+            > 1 Maand </button>
+
+            <button id="six_months" @click="updateData('six_months')"
+              :class="{ active: selection === 'six_months' }"
+            > 6 Maanden </button>
+
+            <button id="one_year" @click="updateData('one_year')"
+              :class="{ active: selection === 'one_year' }"
+            > 1 Jaar </button>
+
+            <button id="ytd" @click="updateData('ytd')"
+              :class="{ active: selection === 'ytd' }"
+            > Dit jaar </button>
+
+            <button id="all" @click="updateData('all')"
+              :class="{ active: selection === 'all' }"
+            > Alles </button>
+          </div>
+
+          <div class="toolbar">
+            <button  id="werkelijk" @click="toggleSerie('werkelijk')"
+            > Werkelijk </button>
+            <button  id="gevoel" @click="toggleSerie('gevoel')"
+            > Gevoel </button>
+            <button  id="dauwpunt" @click="toggleSerie('dauwpunt')"
+            > Dauwpunt </button>
+          </div>
+
+          <div id="chart-timeline">
+            <apexchart
+              type="line"
+              width="1000"
+              height="500"
+              ref="chart"
+              :options="chartOptions"
+              :series="series"
+            ></apexchart>
+          </div>
+        </div>
+        <button class="close" @click="close">sluiten</button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import VueApexCharts from 'vue-apexcharts';
+
+Vue.use(VueApexCharts);
+Vue.component('apexchart', VueApexCharts);
+
+export default {
+  data() {
```

Comment:
Deze mock data word niet gebruikt maar komt uit jsonbin.io? Eventueel verwijderen.

Reply:
Geen http-call gevonden in deze file.